### PR TITLE
404 - redirect in 10sec

### PR DESCRIPTION
**Pull request by paulvi**

### CODE REVIEW

File: source/404.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ Use menu above to navigate or go to [home page](/).
 If you noticed any site problem, typos, etc on any page, 
 please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
 
-You will be redirected to main page in 5 second.
+You will be redirected to main page in 10 second.
```
Comment (yasserzamani) [inline source/404.md]: I think it should be plural I.e. seconds.

File: source/404.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ Use menu above to navigate or go to [home page](/).
 If you noticed any site problem, typos, etc on any page, 
 please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
```
Comment (yasserzamani) [inline source/404.md]: Thanks for your time ! IMHO maybe "you can", "you're able to" or "it's possible to" correct ... are better than "please correct" :)

Comment (paulvi) [reply source/404.md]: It is never enough to be polite. But that should be other PR

**Comment by lukaszlenart** [comment]
👍

**Pull request merged**